.github/workflows/build_all.yml

```diff
@@ -575,7 +575,7 @@ jobs:
         id: publish_artifacts
         if: github.event_name == 'push'
         env:
-          DOCKER_REGISTRY: "cwf-test.artifactory.magmacore.org"
+          DOCKER_REGISTRY: "linuxfoundation.jfrog.io/magma-docker-cwf-test"
```

Comment:
I think this should be `magma-docker-cwag-test`

Reply:
See https://linuxfoundation.jfrog.io/ui/repos/tree/General/magma-docker-cwag-test